docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -173,6 +173,7 @@ To test with Python 3 use the following images:
   - fedora32
   - opensuse15
   - ubuntu1804
+  - ubuntu2004
```

Comment:
The following should be removed:

- fedora28
- ubuntu1404
- ubuntu1604

The following should be added to the Python 3 list:

- alpine3
- fedora33

Reply:
Done, thanks, PTAL